charts/opensearch/templates/statefulset.yaml

```diff
@@ -228,12 +228,13 @@ spec:
 {{- if and .Values.persistence.enabled .Values.persistence.enableInitChown }}
       - name: fsgroup-volume
         image: "{{ template "opensearch.dockerRegistry" . }}{{ .Values.persistence.image | default "busybox" }}:{{ .Values.persistence.imageTag | default "latest" }}"
+        imagePullPolicy: "{{ .Values.image.pullPolicy }}"
```

Comment:
Should this be `.Values.persistence.imagePullPolicy`?

Reply:
Not according to line 245/246, the opensearch container imagePullPolicy gets set to Values.image.pullPolicy